### PR TITLE
add splash screen

### PR DESCRIPTION
**Pull request by nguyenvanman**
refactor guide screen

### CODE REVIEW

File: Zen/Activities/SplashActivity.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public class SplashActivity : Activity
+    {
+        protected override void OnResume()
+        {
+            base.OnResume();
+            var startupWork = new Task(SimulateStartup);
+            startupWork.Start();
+        }
+
+        private async void SimulateStartup()
```
Comment (voquanghoa) [inline Zen/Activities/SplashActivity.cs]: Show `MainActivity` immediately, don't let user wait.

File: Zen/Activities/SplashActivity.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+
+namespace Zen.Activities
+{
+    [Activity(Theme = "@style/Zen.Splash", MainLauncher = true, NoHistory = true)]
```
Comment (voquanghoa) [inline Zen/Activities/SplashActivity.cs]: Set Orientation is Portrait. (also check other activity, except the play video activity)

File: Zen/Activities/SplashActivity.cs (hunk 1 above)
Comment (nguyenvanman) [reply Zen/Activities/SplashActivity.cs]: Done

File: Zen/Activities/SplashActivity.cs (hunk 2 above)
Comment (nguyenvanman) [reply Zen/Activities/SplashActivity.cs]: Done

File: Zen/Activities/SplashActivity.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace Zen.Activities
+{
+    [Activity(Theme = "@style/Zen.Splash", MainLauncher = true, NoHistory = true, ScreenOrientation = ScreenOrientation.Portrait)]
+    public class SplashActivity : Activity
+    {
+        protected override void OnResume()
+        {
+            base.OnResume();
+            StartActivity(new Intent(Application.Context, typeof(MainActivity)));
```
Comment (voquanghoa) [inline Zen/Activities/SplashActivity.cs]: Do it in `onCreated`

**Pull request merged**